kolibri/plugins/facility/assets/src/views/FacilityManagementSideNavEntry.vue

```diff
@@ -27,13 +30,72 @@
       url() {
         return urls['kolibri:kolibri.plugins.facility:facility_management']();
       },
+      facilityRoutes() {
+        return {
+          facilityClasses: {
+            text: this.coreString('classesLabel'),
+            route: this.generateNavRoute(FacilityPageNames.CLASS_MGMT_PAGE),
+          },
+          facilityUsers: {
+            text: this.coreString('usersLabel'),
+            route: this.generateNavRoute(FacilityPageNames.USER_MGMT_PAGE),
+          },
+          facilitySettings: {
+            text: this.$tr('settingsLabel'),
+            route: this.generateNavRoute(FacilityPageNames.FACILITY_CONFIG_PAGE),
+          },
+          facilityData: {
+            text: this.$tr('data'),
+            route: this.generateNavRoute(FacilityPageNames.DATA_EXPORT_PAGE),
+          },
+        };
+      },
+    },
+    methods: {
+      generateNavRoute(route) {
+        // if class id
+        let params;
+        if (this.$store.getters.currentFacilityId) {
```

Comment:
This is incomplete and will break navigation in the case that a user is navigating a facility that is not their own.

This appears to be compounding the regression noted here: https://github.com/learningequality/kolibri/issues/10294